packages/path_provider/path_provider_linux/README.md

```diff
@@ -2,36 +2,29 @@
 
 The linux implementation of [`path_provider`].
 
-**Please set your constraint to `path_provider_linux: '>=0.0.y+x <2.0.0'`**
+**Please set your constraint to `path_provider: '>=0.0.y+x <2.0.0'`**
 
 ## Backward compatible 1.0.0 version is coming
 The `path_provider` plugin has reached a stable API, we guarantee that version `1.0.0` will be backward compatible with `0.0.y+z`.
-Please use `path_provider_linux: '>=0.0.y+x <2.0.0'` as your dependency constraint to allow a smoother ecosystem migration.
+Please use `path_provider: '>=0.0.y+x <2.0.0'` as your dependency constraint to allow a smoother ecosystem migration.
 For more details see: https://github.com/flutter/flutter/wiki/Package-migration-to-1.0.0
 
 ## Usage
 
 ### Import the package
```

Comment:
It seems odd to have the section be called "import the package" when that's not actually necessary.

I think this whole thing could just become:
```
## Usage

This package is already included as part of the `path_provider` package dependency, and will
be included when using `path_provider` as normal. You will need to use version 1.6.10 or newer.
```